docs/tests-github.md

```diff
@@ -0,0 +1,52 @@
+---
+category: DevelopInDepth
+
+---
+# Using GitHub Actions
+
+[Github Action](https://github.com/features/actions) makes it easy to automate all your software workflows, now with world-class CI/CD. Build, test, and deploy your code right from GitHub. Make code reviews, branch management, and issue triaging work the way you want.
+
+
+## GitHub Action Scripts
+
+All the GitHub Action build file are locate in `.github/workflows/*.yml`
+
+Each script is triggered by its own conditions, for more details see [Github Action Docs](https://docs.github.com/en/actions)
+
+## Create a new GitHub Action script
+
+To create a new GitHub Action script please read [Quickstart for GitHub Actions](https://docs.github.com/en/actions/quickstart)
+
+We recommend forking the Matomo project and running actions on your own pipeline first. Also, please check the following list before getting started:
+
+### Existing GitHub Action Scripts
```

Comment:
here I only meant to move down the sentence `(New scripts should be documented in the list above.)` but not the entire section.